libs/shared/providers/src/wagmi/hooks.ts

```diff
@@ -17,7 +17,7 @@ import {
 
 import { isNativeCurrency } from './utils';
 
-import type { Token } from '@origin/shared/contracts';
+import type { NativeToken, Token } from '@origin/shared/contracts';
```

Comment:
I don't think having another type for native token is a good idea, conceptually there is no difference between a token and a native token